src/kernel/CMakeLists.txt

```diff
@@ -100,7 +100,13 @@ set_target_properties(bitcoinkernel PROPERTIES
 
 include(GNUInstallDirs)
 install(TARGETS bitcoinkernel
-  RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
-  LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
-  ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+  RUNTIME
+    DESTINATION ${CMAKE_INSTALL_BINDIR}
+    COMPONENT Kernel
```

Comment:
Why call this `Kernel` and not `bitcoinkernel`?

Reply:
For brevity. Do you want me to update it?

Reply:
I don't think it matters really what it is called. As far as I know there is no way to list all available install components from the command line and discover its name. With the targets you can at least do `cmake --build build --target help`, so my idea was that this might make it more discoverable. But a component might also include multiple targets, so I don't think colliding their names here really helps in terms of making it clearer. So in short, no need to change.